portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,11 @@
+package com.google.sps.servlets;
```

Comment:
Hmm these don't seem relevant to the TDD stuff...

Reply:
Oh, yeah. That stuff is from a previous pull request.